### PR TITLE
DOCSP-4384 Add analytics snippet to hosted docs

### PR DESCRIPTION
**Pull request by cbush**
This adds the analytics snippet when the docs are hosted.

### CODE REVIEW

File: config/header.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<script>
+  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
+  analytics.load("aGhVvyxnPWlyP71vVl9ZjGWxAtoVGLXX");
+  analytics.page({
+    project: 'stitch-android-sdk',
+    branch: 'v4.2',
```
Comment (adamchel) [inline config/header.html]: [q] is this the correct branch?

File: config/header.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<script>
+  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
```
Comment (adamchel) [inline config/header.html]: [q] is 4.1.0 the google analytics snippet version or our SDK version? the Java SDK is on 4.1.4, and will be 4.2.0 when we release?

Comment (cbush) [reply config/header.html]: That's the snippet version, confusingly enough.

File: config/header.html (hunk 1 above)
Comment (cbush) [reply config/header.html]: If the next release is 4.2, I'd say so.

**Pull request merged**